### ISSUE TITLE
firefox - w not working currectly in visual mode before selection.

### ISSUE THREAD

**Issue by rnmhdn** [philc/vimium#3075]
In the visual mode (after pressing 'v') before starting the selection by pressing 'v' again, pressing 'w' takes the cursor one letter ahead instead of one word(same behavior as 'l') however it works fine after starting the selection.

**Comment by smblott-github** [comment]
OK, I think you're referring to what's known as caret mode (when the selection is always just a single character).

You're right, this should be fixed.  Help wanted!

**Comment by muratg** [comment]
Yes, this would be nice. "b" seems to work fine, but some of the others are broken.
